app/metal-controller-manager/api/v1alpha1/server_types.go

```diff
@@ -100,13 +100,16 @@ type ServerSpec struct {
 
 // ServerStatus defines the observed state of Server.
 type ServerStatus struct {
-	Ready bool `json:"ready,omitempty"`
-	InUse bool `json:"inUse,omitempty"`
+	Ready     bool `json:"ready"`
+	InUse     bool `json:"inUse"`
+	NeedsWipe bool `json:"needsWipe"`
```

Comment:
I want to propose `NeedsReset`. Reset sounds better than wipe IMHO. Not a huge deal, I can live with this if we think it is more accurate.

Reply:
Not a big deal, but probably having it inverse - like IsClean might work better as default value is more safe

Reply:
My point was not the name, but rather default value being "dirty" or "needs wipe" vs. default value being "clean". This is more safe.

Reply:
Yeah, I'm just indicating that in this case, it may be worth that non-default valuation.